### PR TITLE
Wizard: language and dialog updates

### PR DESCRIPTION
**Pull request by notmike-5**
#2211 
A more perfected solution would dynamically assign dialog
only when wallet creation from hw device is confirmed.

### CODE REVIEW

**Comment by Realchacal** [comment]
1.) Error in proposed line 66: ("quick") should be "  (quick)" I guess.

2.) Proposed line 125: It's wrong to say that in advanced mode "the blockchain is downloaded to your computer." Because you have all options.

**Comment by notmike-5** [comment]
> 1.) Error in proposed line 66: ("quick") should be " (quick)" I guess.
> 
> 2.) Proposed line 125: It's wrong to say that in advanced mode "the blockchain is downloaded to your computer." Because you have all options.

Was just going off the language suggested in #2208. Happy to make whatever changes. So that we can be as clear as possible, is it the case that:

Simple (quick): connects the wallet to a remote node upon which it relies for verifications. The result being reduced privacy.

Simple (quick + private): connects temporarily to a remote node(s) to download the chain, and upon completion of that task disconnects from the remote node, operating instead as a private node.

Advanced: Downloads the blockchain (not sure how this is differentiated from above), and includes the extra mining, message verification, and etc features?

File: wizard/WizardModeSelection.qml [hunk 1]
```diff
@@ -63,10 +63,10 @@ Rectangle {
             WizardMenuItem {
                 opacity: appWindow.persistentSettings.nettype == 0 ? 1.0 : 0.5
                 Layout.topMargin: 20
-                headerText: qsTr("Simple mode") + translationManager.emptyString
+                headerText: qsTr("Simple mode") + ("quick") + translationManager.emptyString
```
Comment (selsta) [inline wizard/WizardModeSelection.qml]:
```suggestion
                headerText: qsTr("Simple mode") + " (quick)" + translationManager.emptyString
```

File: wizard/WizardModeSelection.qml [hunk 2]
```diff
@@ -121,7 +121,7 @@ Rectangle {
             }
 
             WizardMenuItem {
-                headerText: qsTr("Advanced mode") + translationManager.emptyString
+                headerText: qsTr("Advanced mode") + ("maximum privacy") + translationManager.emptyString
```
Comment (selsta) [inline wizard/WizardModeSelection.qml]:
```suggestion
                headerText: qsTr("Advanced mode") + " (maximum privacy)" + translationManager.emptyString
```

File: wizard/WizardAskPassword.qml [hunk 3]
```diff
@@ -83,7 +83,7 @@ ColumnLayout {
 
     WizardHeader{
         title: qsTr("Give your wallet a password") + translationManager.emptyString
-        subtitle: qsTr("This password cannot be recovered. If you forget it then the wallet will have to be restored from its 25 word mnemonic seed.") + translationManager.emptyString
+        subtitle: qsTr("This password cannot be recovered. If you forget it then the wallet will have to be restored from its 25 word mnemonic seed or your hardware wallet device.") + translationManager.emptyString
```
Comment (selsta) [inline wizard/WizardAskPassword.qml]: I think it would be better to make this context sensitive, e.g. when restoring from a hardware device it only says

> If you forget it then the wallet will have to be restored from your hardware wallet device. 

and vice versa.

Can you revert these changes? I’ll do this in a separate PR.

Comment (notmike-5) [reply wizard/WizardAskPassword.qml]: You got it. Do you know if there is some existing flag that indicates when a wallet is being generated from a hardware device?

Comment (selsta) [reply wizard/WizardAskPassword.qml]: wizardController.walletOptionsIsRecoveringFromDevice

**Comment by Realchacal** [comment]
> > 1.) Error in proposed line 66: ("quick") should be " (quick)" I guess.
> > 2.) Proposed line 125: It's wrong to say that in advanced mode "the blockchain is downloaded to your computer." Because you have all options.
> 
> Was just going off the language suggested in #2208. Happy to make whatever changes. So that we can be as clear as possible, is it the case that:
> 
> Simple (quick): connects the wallet to a remote node upon which it relies for verifications. The result being reduced privacy.
> 
> Simple (quick + private): connects temporarily to a remote node(s) to download the chain, and upon completion of that task disconnects from the remote node, operating instead as a private node.
> 
> Advanced: Downloads the blockchain (not sure how this is differentiated from above), and includes the extra mining, message verification, and etc features?

Please stick exactly to the wording as proposed in #2208 when making a commit claiming to be like #2208. If you want to propose another wording, please discuss it in #2208.

File: wizard/WizardModeSelection.qml [hunk 4]
```diff
@@ -57,16 +57,16 @@ Rectangle {
...
                 title: qsTr("Mode selection.") + translationManager.emptyString
-                subtitle: qsTr("Please select the statement that best matches you.") + translationManager.emptyString
+                subtitle: qsTr("Please select the statement that best matches your desired setup.") + translationManager.emptyString
             }
 
             WizardMenuItem {
                 opacity: appWindow.persistentSettings.nettype == 0 ? 1.0 : 0.5
                 Layout.topMargin: 20
-                headerText: qsTr("Simple mode") + translationManager.emptyString
+                headerText: qsTr("Simple mode") + " (quick)" + translationManager.emptyString
```
Comment (selsta) [inline wizard/WizardModeSelection.qml]:
```suggestion
                headerText: qsTr("Simple mode (quick)") + translationManager.emptyString
```

File: wizard/WizardModeSelection.qml [hunk 5]
```diff
@@ -93,10 +93,10 @@ Rectangle {
 
             WizardMenuItem {
                 opacity: appWindow.persistentSettings.nettype == 0 ? 1.0 : 0.5
-                headerText: qsTr("Simple mode") + " (bootstrap)" + translationManager.emptyString
+                headerText: qsTr("Simple mode") + " (quick + privacy)" + translationManager.emptyString
```
Comment (selsta) [inline wizard/WizardModeSelection.qml]:
```suggestion
                headerText: qsTr("Simple mode (quick + privacy)") + translationManager.emptyString
```

File: wizard/WizardModeSelection.qml [hunk 6]
```diff
@@ -121,8 +121,8 @@ Rectangle {
             }
 
             WizardMenuItem {
-                headerText: qsTr("Advanced mode") + translationManager.emptyString
-                bodyText: qsTr("Includes extra features like mining and message verification. The blockchain is downloaded to your computer.") + translationManager.emptyString
+                headerText: qsTr("Advanced mode") + " (maximum privacy)" + translationManager.emptyString
```
Comment (selsta) [inline wizard/WizardModeSelection.qml]:
```suggestion
                headerText: qsTr("Advanced mode (maximum privacy)") + translationManager.emptyString
```
